src/aics-image-viewer/components/AxisClipSliders/index.tsx

```diff
@@ -71,183 +88,184 @@ const SliderRow: React.FC<SliderRowProps> = ({ label, vals, valsReadout = vals,
   );
 };
 
-interface AxisClipSlidersProps {
+type PlaySliderRowProps = {
+  label: string;
+  val: number;
+  max: number;
+  playing: boolean;
+  entireAxisLoaded?: boolean;
+  onPlayPause: (play: boolean) => void;
+  onChange?: (values: number) => void;
+  onStart?: () => void;
+  onEnd?: () => void;
+};
+
+/** Wrapper around `SliderRow` that adds a play button and accounts for the case where not all of an axis is loaded */
+const PlaySliderRow: React.FC<PlaySliderRowProps> = (props) => {
+  // In partially-loaded axes, stores the displayed value of the slider while the user is sliding it
+  const [valReadout, setValReadout] = useState(props.val);
+  // Tracks when the user is sliding the slider and `valReadout` may have to sub in for props
+  const [sliderHeld, setSliderHeld] = useState(false);
+
+  const wrappedOnChange = useCallback(([val]: number[]) => props.onChange?.(val), [props.onChange]);
+  const wrappedSetValReadout = useCallback(([val]: number[]) => setValReadout(val), []);
+  const wrappedOnStart = useCallback((): void => {
+    setValReadout(props.val);
+    setSliderHeld(true);
+    props.onStart?.();
+  }, [props.onStart]);
+  const wrappedOnEnd = useCallback((): void => {
+    setSliderHeld(false);
+    props.onEnd?.();
+  }, [props.onEnd]);
+
+  return (
+    <>
+      <SliderRow
+        label={props.label}
+        vals={[props.val]}
+        valsReadout={props.entireAxisLoaded || !sliderHeld ? undefined : [valReadout]}
+        max={props.max}
+        onSlide={props.entireAxisLoaded ? wrappedOnChange : wrappedSetValReadout}
+        onChange={props.entireAxisLoaded ? undefined : wrappedOnChange}
+        onStart={wrappedOnStart}
+        onEnd={wrappedOnEnd}
+      />
+      <Tooltip placement="top" title="Play through sequence">
```

Comment:
```suggestion
      <Tooltip placement="top" title="Play through sequence" trigger=["hover", "focus"]>
```
Accessibility for tooltip, this allows tab navigation to also trigger it. (There are a bunch of instances in colorizer where I need to go back and fix this, eep!)

Reply:
The current (quite old) version of Ant used in this project doesn't recognize an array of triggers as a valid type for `trigger`, even though this change does produce the expected behavior? I've had to `@ts-ignore` it for now.

We have an ancient issue out to update to a newer Ant, but never made much progress because upgrading Ant breaks a lot of stuff in ways that are tedious to fix.